src/group_theory/perm/support.lean

```diff
@@ -0,0 +1,477 @@
+/-
+Copyright (c) 2018 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Aaron Anderson, Yakov Pechersky
+-/
+import data.finset.sort
+import data.fintype.basic
+import group_theory.perm.basic
+
+/-!
+# Support of a permutation
+
+## Main definitions
+In the following, `f g : equiv.perm α`.
+* `equiv.perm.disjoint`: two permutations `f` and `g` are `disjoint` iff their `support`
+  are disjoint.
```

Comment:
This sounds like a description of `disjoint_iff_disjoint_support`, rather than a definition of `disjoint`.